lib/ramble/ramble/renderer.py

```diff
@@ -139,7 +187,28 @@ def render_objects(self, variables, matrices=None):
             for names, vectors in zip(matrix_variables, matrix_vectors):
                 for obj_idx, entry in enumerate(itertools.product(*vectors)):
                     for name_idx, name in enumerate(names):
-                        matrix_objects[obj_idx][name] = entry[name_idx]
+                        if name in defined_zips.keys():
+                            # Replace the zip name with the constituent variables
+                            for zip_var in defined_zips[name]['vars']:
+                                matrix_objects[obj_idx][zip_var] = \
+                                    defined_zips[name]['vars'][zip_var][entry[name_idx]]
+
+                            # Consume the defined zip
+                            consumed_zips.add(name)
+                        else:
+                            matrix_objects[obj_idx][name] = entry[name_idx]
+
+        # Remove all consumed zips and
```

Comment:
any reason you broken the command lines like this?

Reply:
Fixed!